loadMon.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+trigger=5.00
+load=`cat /proc/loadavg | awk '{print $1}'`
+response=`echo | awk -v T=$trigger -v L=$load 'BEGIN{if ( L > T){ print "greater"}}'`
+if [[ $response = "greater" ]]
+then
+killall screen
```

Comment:
This seems a bit aggressive. It would kill all screens running on the host machine.
A better solution would be to store the pids of the two screens that we spawn and SIGHUP them.